tasks/xml2json.js

```diff
@@ -28,6 +28,11 @@ module.exports = function(grunt) {
                     id: match[1],
                     text: match[4].replace(/&#13;\n/g, " ").replace(/&#160;/g, " ")
                 };
+
+                if(/\/translations\/Messages_\w{2}_\w{2,4}.xml$/.test(filePath)) {
```

Comment:
Hard to understand, could you please either split into multiple lines or add comments ? 😄